app/styles/_ellipsis.less

```diff
@@ -133,3 +133,8 @@
     z-index: 1;
   }
 }
+
+// static loading... spacing
+.section-loading {
+  margin: 20px 0;
```

Comment:
Just wondering if we really should have a generic margin for the loading message. We usually try to position the loading message exactly where the content displays to avoid content jumping on the page as it loads. So the content would need the same margin, but `class="section-loading"` doesn't make sense on the content.

I'm thinking we should have a bottom margin as part of the header bottom border, then not have top margin on the content on any of those pages. That way we're always consistent.

I do like that we are adding standard classes for this stuff. The `mar-top-*` classes mean we frequently use different margins on different pages.

/cc @rhamilto